readthedocs/restapi/urls.py

```diff
@@ -65,7 +65,6 @@
 function_urls = [
     url(r'embed/', core_views.embed, name='embed'),
```

Comment:
This one isn't used either, and it doesn't work

Reply:
Yea, we should kill this. We have the `embed` in our `ext` repo that actually does work.

Reply:
I opened https://github.com/rtfd/readthedocs.org/issues/4768